js/src/popover.js

```diff
@@ -90,6 +90,29 @@ class Popover extends Tooltip {
     return this.getTitle() || this._getContent()
   }
 
+  getTipElement() {
+    if (this.tip) {
+      return this.tip
+    }
+
+    const element = document.createElement('div')
+    element.innerHTML = this._config.template
```

Comment:
This should respect the sanitizer like we do elsewhere.

Reply:
Yes that is done in the tooltip class in the `_getConfig` method :thinking: